config/puma.rb

```diff
@@ -0,0 +1,21 @@
+# https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server
+
+workers Integer(ENV['WEB_CONCURRENCY'] || 1)
```

Comment:
Doesn't seem like you have `newrelic_rpm` included in [the Gemfile](https://github.com/codeforamerica/ohana-api/blob/master/Gemfile)...am I missing something? Might be why you're not seeing the issue :wink: